### DISCUSSION TITLE
Otter compatibility with custom layouts

### DISCUSSION THREAD

**Discussion by JohnPixle** [Codeinwp/otter-blocks#1519] [Ideas & Feedback]
Hey all, I am creating this as a master issue to keep track of all things that will need to be addressed for Custom layouts compatibility. Will be updating this one accordingly. 

- [x] #1269
- [x] #1272
- [x] #1270
- [ ] #1341
- [ ] #1436

**Comment by HardeepAsrani** [comment]
This will be helpful. Let us know all you can find, I'll start looking into them once we're out with the current release.

**Comment by HardeepAsrani** [comment]
Hey @JohnPixle @mghenciu Let us know if there are any more issues that you found apart from the ones listed, with both Dynamic Conditions and in general with Custom Layout/FSE + Otter.

**Comment by mghenciu** [comment]
For this stage @HardeepAsrani , I think **first we need to have the main requirements checked, meaning that Otter works properly in a Custom Layout context** - and latter go into a testing phase with different Custom Layout conditions, etc.

 For example, what I did first is to copy a bunch of Otter Patterns and add them into a Custom Layout. As seen in the video below, those look have some issues with the Container box, so that's one thing we need to fix first:

https://user-images.githubusercontent.com/52494172/212062256-e5f9ca85-2fb4-4a0f-91b8-af3bb242fc2c.mov
